arm-dependencies.sls

```diff
@@ -0,0 +1,52 @@
+arm-dependencies:
+  pkg.installed:
+    - pkgs:
+      - g++-aarch64-linux-gnu
+      - g++-arm-linux-gnueabihf
+
+{% set targets = [{ 'name': 'arm-linux-gnueabihf', 
+                    'file': 'https://servo-rust.s3.amazonaws.com/ARM/v1/armhf-trusty-libs.tgz',
```

Comment:
Use the (to-be-added) `version` and `local_name` variables to create these URLs where it's needed in the archive.extracted state, instead of setting them here.